package.json

```diff
@@ -9,8 +9,12 @@
     "bouttime-server": "bin/bouttime-server"
   },
   "scripts": {
+    "build": "bower install && gulp",
+    "start-dev": "nodemon --watch dist/ --delay 5 bin/bouttime-server",
```

Comment:
The delay is a bit of a workaround as a large number of changes caught by gulp sees the server bouncing up and down.